Fix link failures observed in riscv64 builds for questing and resolute:

  ld.lld: error: undefined hidden symbol: ARGBAttenuateRow_RVV
  ld.lld: error: undefined hidden symbol: CopyRow_RVV
  ld.lld: error: undefined hidden symbol: I400ToARGBRow_RVV
  ld.lld: error: undefined hidden symbol: I422AlphaToARGBRow_RVV
  ld.lld: error: undefined hidden symbol: I422ToARGBRow_RVV
  ld.lld: error: undefined hidden symbol: I444ToARGBRow_RVV
  ld.lld: error: undefined hidden symbol: InterpolateRow_RVV
  ld.lld: error: undefined hidden symbol: J400ToARGBRow_RVV
  ld.lld: error: undefined hidden symbol: ScaleARGBRowDown2Box_RVV
  ld.lld: error: undefined hidden symbol: ScaleARGBRowDown2Linear_RVV
  ld.lld: error: undefined hidden symbol: ScaleARGBRowDown2_RVV
  ld.lld: error: undefined hidden symbol: ScaleARGBRowDownEvenBox_RVV
  ld.lld: error: undefined hidden symbol: ScaleUVRowDown2Box_RVV
  ld.lld: error: undefined hidden symbol: ScaleUVRowDown2Linear_RVV
  ld.lld: error: undefined hidden symbol: ScaleUVRowDown2_RVV
  ld.lld: error: undefined hidden symbol: ScaleUVRowDown4_RVV
  ld.lld: error: undefined hidden symbol: ScaleUVRowDownEven_RVV
  ld.lld: error: undefined hidden symbol: ScaleUVRowUp2_Bilinear_RVV
  ld.lld: error: undefined hidden symbol: ScaleUVRowUp2_Linear_RVV
  ld.lld: error: undefined hidden symbol: SplitUVRow_RVV

--- a/media/libyuv/libyuv/libyuv.gypi
+++ b/media/libyuv/libyuv/libyuv.gypi
@@ -86,6 +86,7 @@
       'source/row_gcc.cc',
       'source/row_lasx.cc',
       'source/row_lsx.cc',
+      'source/row_rvv.cc',
       'source/row_win.cc',
       'source/scale.cc',
       'source/scale_any.cc',
@@ -94,6 +95,7 @@
       'source/scale_gcc.cc',
       'source/scale_lsx.cc',
       'source/scale_rgb.cc',
+      'source/scale_rvv.cc',
       'source/scale_uv.cc',
       'source/scale_win.cc',
       'source/video_common.cc',
